juju-tutorials/juju-tricks.md

```diff
@@ -21,47 +21,51 @@ If you prefer chatting, visit us on [IRC](https://webchat.freenode.net/#juju).
 Duration: N/A
 
 [note status="Direct reachability"]
-For the commands in this section to work, the targeted units or machines must be directly reachable via `juju ssh`. If, for example, LXCs are using network that is reachable only from the LXC host machine, you will encounter connectivity errors.
+For the commands in this section to work, the targeted units or machines must be directly reachable via `juju ssh`. If, for example, LXDs are using network that is reachable only from the LXD host machine, you will encounter connectivity errors.
 [/note]
 
 ### Restarting units and machines based on their state
 
-To restart units that are in particular state (e.g. `lost` or `down`) use following command and adjust the first part, `state=<TARGET_STATE>`, to match state of the units that you want to restart. The command, as it is, will restart units in state `down`.
+To restart units that are in a particular state (e.g. `lost` or `down`) use the following command and adjust the first part, `state=<TARGET_STATE>`, to match the state of the units that you want to restart. The command, as it is, will restart the units in state `down`.
 
 ```console
-state=down; juju status --format oneline | grep "agent:${state}" | xargs -l4 | column -t | egrep -o '[a-z0-9-]+/[0-9]+' | sort -u | xargs -P4 -I@ bash -c 'unit=@; juju ssh $unit "sudo service jujud-unit-${unit/\//-} restart"'
+state=down; juju status --format oneline | grep "agent:${state}" | egrep -o '[a-z0-9-]+/[0-9]+' | sort -u | xargs -P4 -I@ bash -c 'unit=@; juju ssh $unit "sudo service jujud-unit-${unit/\//-} restart"'
 ```
 
-To restart machine in particular state (including LXCs) use the following command and adjust the first part, `state=<TARGET_STATE>`, to match state of the machines that you want to restart. The command, as it is, will restart machines in state `down`.
+To restart machine in a particular state (including LXDs) use the following command and adjust the first part, `state=<TARGET_STATE>`, to match the state of the machines that you want to restart. The command, as it is, will restart the machines in state `down`.
```

Comment:
"To restart machines in a particular..." or "To restart a machine in a particular..."